changelog.d/15891.feature

```diff
@@ -0,0 +1 @@
+Implements a task scheduler for	resumable potentially long running tasks.
```

Comment:
I don't think this needs to be a feature -- it is something internally used by Synapse; admins and end-users don't care.

We could probably just combine this with the other PR?